plugins/auth-node/src/oauth/OAuthCookieManager.ts

```diff
@@ -16,8 +16,9 @@
 
 import { CookieOptions, Request, Response } from 'express';
 import { CookieConfigurer } from '../types';
+import { HumanDuration, durationToMilliseconds } from '@backstage/types';
 
-const THOUSAND_DAYS_MS = 1000 * 24 * 60 * 60 * 1000;
+const FOUR_HUNDRED_DAYS_MS = 400 * 24 * 60 * 60 * 1000;
```

Comment:
Sure this might be the actual max in most cases, but does it hurt to keep it at 1000? The intention here is just for it to last for a long time. I'd rather keep as is tbh

Reply:
Sure can do that, either way works. The value will be [capped by the browser](https://developer.chrome.com/blog/cookie-max-age-expires#example) anyways 👍